# French translation for sbackup
# Copyright (c) 2007 Rosetta Contributors and Canonical Ltd 2007
# This file is distributed under the same license as the sbackup package.
# FIRST AUTHOR <EMAIL@ADDRESS>, 2007.
#
msgid ""
msgstr ""
"Project-Id-Version: sbackup\n"
"Report-Msgid-Bugs-To: \n"
"POT-Creation-Date: 2013-01-10 22:47+0100\n"
"PO-Revision-Date: 2013-01-11 16:14+0000\n"
"Last-Translator: Olivier Maury <Unknown>\n"
"Language-Team: French <fr@li.org>\n"
"MIME-Version: 1.0\n"
"Content-Type: text/plain; charset=UTF-8\n"
"Content-Transfer-Encoding: 8bit\n"
"X-Launchpad-Export-Date: 2013-01-12 05:07+0000\n"
"X-Generator: Launchpad (build 16420)\n"

#: src/sbackup/backupproc.py:169 src/sbackup/pkginfo.py:85
#, python-format
msgid "SBackup Daemon <%(login)s@%(hostname)s>"
msgstr "Démon Sbackup <%(login)s@%(hostname)s>"

#: src/sbackup/backupproc.py:173
#, python-format
msgid "[SBackup] [%(profile)s] Report of %(date)s"
msgstr "[Sbackup] [%(profile)s] Rapport du %(date)s"

#: src/sbackup/backupproc.py:178 src/sbackup/core/profile_handler.py:282
msgid "No log file specified."
msgstr "Aucun fichier journal indiqué."

#: src/sbackup/backupproc.py:183
msgid "Unable to find log file."
msgstr "Impossible de trouver le fichier journal."

#: src/sbackup/backupproc.py:251
msgid ""
"Critical Error: No configuration file for the default profile was found!\n"
"\n"
"Now continue processing remaining profiles."
msgstr ""
"Erreur critique : aucun fichier de configuration trouvé pour le profil par "
"défaut !\n"
"Continuer maintenant à traiter les profils restants."

#: src/sbackup/backupproc.py:302
msgid "Backup was canceled by user."
msgstr "Sauvegarde annulée par l'utilisateur."

#: src/sbackup/backupproc.py:307
msgid "An error occurred during the cancellation of backup:"
msgstr "Une erreur est survenue durant l'annulation de la sauvegarde:"

#: src/sbackup/backupproc.py:313
msgid "An error occurred during the backup:"
msgstr "Une erreur est survenue durant la sauvegarde :"

#: src/sbackup/backupproc.py:330
msgid "Error when sending email:"
msgstr "Erreur pendant l'envoi de courriel."

#: src/sbackup/backupproc.py:361
msgid "The following error(s) occurred before:"
msgstr "Les erreurs suivantes ont eu lieu avant:"

#: src/sbackup/backupproc.py:532
#, python-format
msgid ""
"Backup is not being started.\n"
"%s"
msgstr ""
"La sauvegarde n'a pas encore démarré.\n"
"%s"

#: src/sbackup/pkginfo.py:42
msgid ""
"This is a user friendly backup solution for common desktop needs. If you "
"want to help by submitting bugs, by translating or anything else please "
"visit our website."
msgstr ""
"Voici une solution de sauvegarde conviviale pour les usages courants en "
"bureautique. Si vous souhaitez aider en soumettant des bugs, en traduisant "
"ou quoique que se soit d'autre, merci de visiter notre site internet."

#: src/sbackup/pkginfo.py:45
msgid "Maintainers:"
msgstr "Mainteneurs :"

#: src/sbackup/pkginfo.py:49
msgid "Former maintainers:"
msgstr "Anciens contributeurs :"

#: src/sbackup/pkginfo.py:53
msgid "Contributors:"
msgstr "Contributeurs :"

#: src/sbackup/pkginfo.py:70
msgid "translator-credits"
msgstr ""
"Launchpad Contributions:\n"
"  Djiboun https://launchpad.net/~djiboun\n"
"  Eric Noel https://launchpad.net/~profnoel\n"
"  François Tissandier https://launchpad.net/~baloo\n"
"  Gwendal Jabot https://launchpad.net/~gwendal-jabot\n"
"  Jean-Peer Lorenz https://launchpad.net/~peer.loz\n"
"  LEROY Jean-Christophe https://launchpad.net/~celtic2-deactivatedaccount\n"
"  NSV https://launchpad.net/~nsv\n"
"  Olivier Maury https://launchpad.net/~olivier-maury\n"
"  Olivier Tétard https://launchpad.net/~otetard\n"
"  Pierre Slamich https://launchpad.net/~pierre-slamich\n"
"  gilles chaon https://launchpad.net/~g-chaon\n"
"  manu https://launchpad.net/~manu.pintor"

#: src/sbackup/ar_backend/tar.py:109 src/sbackup/ar_backend/tar.py:167
#: src/sbackup/ar_backend/tar.py:213
msgid "Invalid archive type."
msgstr "Type d'archive invalide"

#: src/sbackup/ar_backend/tar.py:123
#, python-format
msgid "Effective path for `%s` is not available"
msgstr "Le chemin utilisé pour '%s' n'est pas disponible"

#: src/sbackup/ar_backend/tar.py:295
#, python-format
msgid "Unable to get effective path for `%s`"
msgstr "Impossible d'obtenir le chemin utilisé pour '%s'"

#: src/sbackup/ar_backend/tar.py:344
msgid ""
"For the moment split functionality is not compatible with compress option."
msgstr ""
"Pour le moment la fonctionnalité scinder n'est pas compatible avec l'option "
"compression."

#: src/sbackup/ar_backend/tar.py:353 src/sbackup/ar_backend/tar.py:414
#: src/sbackup/core/profile_handler.py:292
#, python-format
msgid "Unable to change permissions for file '%s'."
msgstr "Impossible de changer la permission du fichier '%s'."

#: src/sbackup/ar_backend/tar.py:362
#, python-format
msgid "Unable to remove temporary file `%(file)s`: %(error)s"
msgstr "Impossible d'effacer le fichier temporaire `%(file)s`: %(error)s"

#: src/sbackup/ar_backend/tar.py:378
msgid "Launching TAR to make incremental backup."
msgstr "Lancement de TAR pour faire une sauvegarde incrémentielle."

#: src/sbackup/ar_backend/tar.py:407
msgid "Unable to find the SNAR file to make an incremental backup."
msgstr ""
"Impossible de trouver le fichier SNAR pour faire une sauvegarde "
"incrémentielle."

#: src/sbackup/ar_backend/tar.py:408
msgid "Falling back to full backup."
msgstr "Retour à une sauvegarde complète."

#: src/sbackup/ar_backend/tar.py:450
msgid "Launching TAR to make a full backup."
msgstr "Lancement de TAR pour faire une sauvegarde complète."

#: src/sbackup/ar_backend/tar.py:517
msgid "Leading '/' from member names were removed."
msgstr "La barre oblique '/' précédant le nom du membre a été supprimée."

#: src/sbackup/ar_backend/tar.py:519
msgid "Leading '/' from hard link targets were removed."
msgstr "La barre oblique '/' précédant le lien de la cible a été supprimée."

#: src/sbackup/ar_backend/tar.py:528
#, python-format
msgid "TAR returned a message: %s"
msgstr "TAR a envoyé le message: %s"

#: src/sbackup/ar_backend/tar.py:530
#, python-format
msgid ""
"TAR returned messages:\n"
"%s"
msgstr ""
"TAR a envoyé le message:\n"
"%s"

#: src/sbackup/ar_backend/tar.py:533
msgid "TAR has been finished successfully."
msgstr "TAR terminé avec succès."

#: src/sbackup/ar_backend/tar.py:537
#, python-format
msgid "TAR returned a warning: %s"
msgstr "TAR a envoyé l'avertissement: %s"

#: src/sbackup/ar_backend/tar.py:539
#, python-format
msgid ""
"TAR returned warnings:\n"
"%s"
msgstr ""
"TAR a envoyé l'avertissement:\n"
"%s"

#: src/sbackup/ar_backend/tar.py:542
msgid "TAR returned warnings but has been finished successfully."
msgstr "TAR a envoyé un avertissement, mais a complété avec succès."

#: src/sbackup/ar_backend/tar.py:546
msgid "Unable to finish successfully. TAR terminated with errors."
msgstr "Impossible d'achever avec succès. TAR a terminé avec des erreurs."

#: src/sbackup/ar_backend/tar.py:727
#, python-format
msgid "Unable to terminate backend process: %s"
msgstr "Impossible d'achever le processus en arrière-plan: %s"

#: src/sbackup/ar_backend/tar.py:760
msgid "Included"
msgstr "Inclus"

#: src/sbackup/ar_backend/tar.py:760
msgid "Not changed"
msgstr "Inchangé"

#: src/sbackup/ar_backend/tar.py:760
msgid "Directory"
msgstr "Répertoire"

#: src/sbackup/ar_backend/tar.py:788
msgid "Line must be a string"
msgstr "La ligne doit être une chaîne de caractères"

#: src/sbackup/ar_backend/tar.py:804
msgid "Dumpdir inconsistancy : 'filename' is empty"
msgstr "Inconsistance de 'Dumpdir' : 'nom fichier' est vide"

#: src/sbackup/ar_backend/tar.py:817
msgid "Dumpdir inconsistancy : 'control' is empty"
msgstr "Inconsistance de 'Dumpdir' : 'control' est vide"

#: src/sbackup/ar_backend/tar.py:890
#, python-format
msgid "File '%s' does not exist."
msgstr "Le fichier '%s' n'existe pas."

#: src/sbackup/ar_backend/tar.py:1020 src/sbackup/ar_backend/tar.py:1054
#: src/sbackup/ar_backend/tar.py:1154
msgid "The snarfile header is incomplete."
msgstr "L'entête du snarfile est incomplet."

#: src/sbackup/ar_backend/tar.py:1323
msgid "A SnapshotFile is required"
msgstr "Un fichier d'Instantané est requis"

#: src/sbackup/ar_backend/tar.py:1423
#, python-format
msgid "Directory does not exist: %s."
msgstr "Le répertoire n'existe pas:%s."

#: src/sbackup/core/ConfigManager.py:330
#, python-format
msgid "Profile settings are being read from file '%s'."
msgstr "Les réglages du profil sont lus à partir du fichier '%s'."

#: src/sbackup/core/ConfigManager.py:333
#, python-format
msgid ""
"Profile settings are being set to default values. Configuration file is set "
"to '%s'."
msgstr ""
"Les réglages du profil sont faits avec les valeurs par défaut. Le fichier de "
"configuration est '%s'."

#: src/sbackup/core/ConfigManager.py:479
msgid "Option 'target' is missing, aborting."
msgstr "L'option 'cible' est manquante, abandon."

#: src/sbackup/core/ConfigManager.py:547
#, python-format
msgid ""
"Error while processing configuration:\n"
"\n"
"%(error)s in line `%(line)s`.\n"
"\n"
"Please check your configuration file."
msgstr ""
"Erreur pendant le traitement de la configuration :\n"
"\n"
"%(error)s ligne(s) `%(line)s`.\n"
"\n"
"Veuillez vérifier votre fichier de configuration."

#: src/sbackup/core/ConfigManager.py:588 src/sbackup/core/ConfigManager.py:622
#, python-format
msgid ""
"Unable to evaluate '%(parameter)s' as a dictionary (value got = '%(value)r')."
msgstr ""
"Impossible d'évaluer '%(parameter)s' comme un dictionnaire (valeur obtenue = "
"'%(value)r')."

#: src/sbackup/core/ConfigManager.py:614
#, python-format
msgid ""
"Couldn't evaluate '%(parameter)s' as a dictionary (value got = '%(value)r' )"
msgstr ""
"Impossible d'évaluer '%(parameter)s' comme un dictionnaire (valeur obtenue = "
"'%(value)r' )"

#: src/sbackup/core/ConfigManager.py:638
msgid "You must provide a dictionary."
msgstr "Vous devez fournir un dictionnaire."

#: src/sbackup/core/ConfigManager.py:755
#, python-format
msgid "Log output for [%(profile)s] is directed to file '%(file)s'."
msgstr "Le journal pour [%(profile)s] sera écrit dans '%(file)s'."

#: src/sbackup/core/ConfigManager.py:762
#, python-format
msgid "Log output for [%s] is not directed into a file."
msgstr "La journal pour [%s] n'est pas écrit dans un fichier."

#: src/sbackup/core/ConfigManager.py:781
#, python-format
msgid "Unable to rename log file '%(src)s'->'%(dst)s': %(err)s"
msgstr ""
"Impossible de renommer le fichier journal '%(src)s'->'%(dst)s': %(err)s"

#: src/sbackup/core/ConfigManager.py:801
#, python-format
msgid "The config file '%s' couldn't be read!"
msgstr "Le fichier de configuration '%s' ne peut être lu!"

#: src/sbackup/core/ConfigManager.py:813
#, python-format
msgid "section [%(section)s] in '%(configfile)s' should not exist, aborting"
msgstr ""
"La section [%(section)s] dans '%(configfile)s' ne devrait pas exister. "
"Abandon."

#: src/sbackup/core/ConfigManager.py:895
msgid "No schedule defined in configuration file. Probing from filesystem."
msgstr ""
"Pas de calendrier défini dans le fichier de configuration. Recherche dans le "
"système de fichier."

#: src/sbackup/core/ConfigManager.py:988
#: src/sbackup/core/ConfigManager.py:1000
msgid "The config file is not set yet into this ConfigManager"
msgstr ""
"Le fichier de configuration n'est pas encore précisé dans ce ConfigManager"

#: src/sbackup/core/ConfigManager.py:1163
msgid "No receiver set."
msgstr "Pas de destination défini."

#: src/sbackup/core/ConfigManager.py:1165
msgid "No SMTP server set."
msgstr "Pas de serveur SMTP défini."

#: src/sbackup/core/ConfigManager.py:1168
msgid "Username set but no password specified."
msgstr "Nom d'utilisateur défini, mais pas le mot de passe."

#: src/sbackup/core/ConfigManager.py:1171
msgid "Password set but no username specified."
msgstr "Mot de passe défini, mais pas le nom d'utilisateur."

#: src/sbackup/core/ConfigManager.py:1174
msgid ""
"A certificate and key file is given while SSL option (smtptls=1) is not "
"set.\n"
"Select SSL in order to use Certificate and Key."
msgstr ""
"Un certificat et un fichier clé sont fournis alors que l'option SSL "
"(smtptls=1) n'est pas activé.\n"
"Sélectionnez SSL afin d'utiliser le certificat et la clé."

#: src/sbackup/core/ConfigManager.py:1178
msgid ""
"When specifying a SSL certificate or key file, a key file resp. certificate "
"is mandatory."
msgstr ""
"Lors de la spécification d'un certificat SSL ou d'un fichier clé, un fichier "
"clé resp. certifié est obligatoire."

#: src/sbackup/core/ConfigManager.py:1373
msgid "Default Profile"
msgstr "Profil par défaut"

#: src/sbackup/core/ConfigManager.py:1374
msgid "Unknown Profile"
msgstr "Profil inconnu"

#: src/sbackup/core/ConfigManager.py:1428
msgid "Unlimited"
msgstr "Illimité"

#: src/sbackup/core/ConfigManager.py:1429
msgid "100 MiB"
msgstr "100 Mo"

#: src/sbackup/core/ConfigManager.py:1430
msgid "250 MiB"
msgstr "250 Mo"

#: src/sbackup/core/ConfigManager.py:1431
msgid "650 MiB"
msgstr "650 Mo"

#: src/sbackup/core/ConfigManager.py:1432
msgid "2 GiB (FAT16)"
msgstr "2 Go (FAT16)"

#: src/sbackup/core/ConfigManager.py:1433
msgid "4 GiB (FAT32)"
msgstr "4 Go (FAT32)"

#: src/sbackup/core/ConfigManager.py:1434 src/sbackup/ui/configgui.py:279
#: src/sbackup/ui/configgui.py:1418
msgid "Custom"
msgstr "Personnalisé"

#: src/sbackup/core/ConfigManager.py:1436
msgid "MP3 Music"
msgstr "Musique MP3"

#: src/sbackup/core/ConfigManager.py:1437
msgid "AVI Video"
msgstr "Vidéo AVI"

#: src/sbackup/core/ConfigManager.py:1438
#: src/sbackup/core/ConfigManager.py:1439
msgid "MPEG Video"
msgstr "Vidéo MPEG"

#: src/sbackup/core/ConfigManager.py:1440
msgid "Matrjoshka Video"
msgstr "Vidéo Matroska"

#: src/sbackup/core/ConfigManager.py:1441
msgid "OGG Multimedia container"
msgstr "Conteneur multimédia OGG"

#: src/sbackup/core/ConfigManager.py:1442
msgid "CD Images"
msgstr "Images CD"

#: src/sbackup/core/RestoreManager.py:80
#, python-format
msgid "File '%s' not found in the backup snapshot files list"
msgstr "Le fichier '%s'  est absent de la liste de fichiers de l'instantané"

#: src/sbackup/core/RestoreManager.py:82
#, python-format
msgid ""
"File '%(filename)s' not found in snapshot's [%(snapshotname)s] files list, "
"Skipped."
msgstr ""
"Le fichier '%(filename)s' est absent de la liste de fichiers de l'instantané "
"[%(snapshotname)s], Ignorée."

#: src/sbackup/core/SnapshotManager.py:114
#, python-format
msgid "Snapshot '%s' not found "
msgstr "L'instantané '%s' non trouvé. "

#: src/sbackup/core/SnapshotManager.py:210
#, python-format
msgid "Invalid snapshot `%(name)s` found: Name of snapshot not valid."
msgstr ""
"L'instantané `%(name)s` est invalide raison: Le nom de l'instantané n'est "
"pas valide."

#: src/sbackup/core/SnapshotManager.py:213
#, python-format
msgid "Invalid snapshot `%(name)s` found: %(error_cause)s."
msgstr "L'instantané `%(name)s` est invalide raison: %(error_cause)s."

#: src/sbackup/core/SnapshotManager.py:286
msgid "The snarfile header is incomplete !"
msgstr "L'entête du fichier SNAR est incomplète !"

#: src/sbackup/core/SnapshotManager.py:429
#, python-format
msgid ""
"Snapshot '%s' is already Full, nothing to do (not changing it to full)."
msgstr "L'instantané '%s' est déjà plein, abandon de l'opération."

#: src/sbackup/core/profile_handler.py:101
#, python-format
msgid "Running of hooks: %s"
msgstr ""

#: src/sbackup/core/profile_handler.py:111
msgid "Preparation of backup process"
msgstr "Préparation du processus de sauvegarde"

#: src/sbackup/core/profile_handler.py:146
#, python-format
msgid "Snapshot '%(name)s' is being made."
msgstr "Instantané '%(name)s' en cours de création."

#: src/sbackup/core/profile_handler.py:154
#, python-format
msgid "Setting Base to '%(value)s'."
msgstr "Réglage de la Base à '%(value)s'."

#: src/sbackup/core/profile_handler.py:163
msgid "Setting packages File."
msgstr "Réglage des fichiers de sauvegarde."

#: src/sbackup/core/profile_handler.py:169
msgid "Problem when setting the packages list: "
msgstr "Il y a un problème lors du réglage de la liste des paquets. "

#: src/sbackup/core/profile_handler.py:173
msgid "Setting Excludes File."
msgstr "Réglage des exclusions de fichiers."

#: src/sbackup/core/profile_handler.py:181
#, python-format
msgid "Setting compression format to `%s`"
msgstr "Réglage du format de compression à `%s`"

#: src/sbackup/core/profile_handler.py:188
#, python-format
msgid "Setting size of archive chunks to %s"
msgstr "Réglage de la taille des parties d'archive à %s"

#: src/sbackup/core/profile_handler.py:195
msgid "Option 'Follow symbolic links' is enabled."
msgstr "L'option 'suivre les liens symboliques' est activée."

#: src/sbackup/core/profile_handler.py:197
msgid "Option 'Follow symbolic links' is disabled."
msgstr "L'option 'Suivre les liens symboliques' est désactivée."

#: src/sbackup/core/profile_handler.py:206
msgid "Snapshot is being committed"
msgstr "Un instantané est en cours"

#: src/sbackup/core/profile_handler.py:220
#, python-format
msgid "Error while purging old snapshots: %s"
msgstr "Erreur pendant le nettoyage des anciens instantanés: %s"

#: src/sbackup/core/profile_handler.py:222
msgid "Backup process finished."
msgstr "Le processus de sauvegarde est terminé."

#: src/sbackup/core/profile_handler.py:229
msgid "Inspect file system and collect file infos"
msgstr ""
"Inspection du système de fichiers et collecte des informations sur les "
"fichiers"

#: src/sbackup/core/profile_handler.py:240
msgid "Summary of backup"
msgstr "Résumé de la sauvegarde"

#: src/sbackup/core/profile_handler.py:241
#, python-format
msgid "Number of directories: %s."
msgstr "Nombre de répertoires : %s."

#: src/sbackup/core/profile_handler.py:242
#, python-format
msgid "Total number of files: %s."
msgstr "Nombre total de fichiers : %s."

#: src/sbackup/core/profile_handler.py:243
#, python-format
msgid "Number of symlinks: %s."
msgstr "Nombre de liens symboliques : %s."

#: src/sbackup/core/profile_handler.py:244
#, python-format
msgid "Number of files included in snapshot: %s."
msgstr "Nombre de fichiers inclus dans l'instantané : %s."

#: src/sbackup/core/profile_handler.py:245
#, python-format
msgid "Number of new files (also included): %s."
msgstr "Nombre de nouveaux fichiers (également inclus) : %s."

#: src/sbackup/core/profile_handler.py:246
#, python-format
msgid "Number of files skipped in incremental snapshot: %s."
msgstr "Nombre de fichiers ignorés dans l'instantané incrémental : %s."

#: src/sbackup/core/profile_handler.py:247
#, python-format
msgid "Number of items forced to be excluded: %s."
msgstr "Nombre de fichiers exclus de force : %s."

#: src/sbackup/core/profile_handler.py:248
#, python-format
msgid "Number of items to be excluded by config: %s."
msgstr "Nombre de fichiers exclus selon la configuration : %s."

#: src/sbackup/core/profile_handler.py:249
#, python-format
msgid "Maximum free size required is '%s'."
msgstr "L'espace libre maximum requis est '%s'."

#: src/sbackup/core/profile_handler.py:255
#, python-format
msgid "Available disk size is '%s'."
msgstr "L'espace disque disponible est '%s'."

#: src/sbackup/core/profile_handler.py:257
#, python-format
msgid ""
"Not enough free space in the target directory for the planned backup (free: "
"%(freespace)s, required: %(neededspace)s)."
msgstr ""
"Il n'y a pas assez d'espace disponible dans le répertoire cible pour la "
"sauvegarde planifiée (libre:%(freespace)s, requis: %(neededspace)s)."

#: src/sbackup/core/profile_handler.py:277
msgid "Unable to copy log. File access is not initialized."
msgstr ""
"Impossible de copier le journal. L'accès au fichier n'est pas initialisé."

#: src/sbackup/core/profile_handler.py:295
#, python-format
msgid "Unable to copy log file: %s"
msgstr "Impossible de copier le fichier journal : %s"

#: src/sbackup/core/profile_handler.py:297
msgid "Unable to find logfile to copy into snapshot."
msgstr ""
"Impossible de trouver le fichier de journal à copier dans l'instantané."

#: src/sbackup/core/profile_handler.py:299
msgid "No snapshot to copy logfile."
msgstr "Aucun instantané pour copier le fichier de rapport."

#: src/sbackup/core/profile_handler.py:306
msgid "Processing of profile was canceled on user request\n"
msgstr "Le traitement du profil a été annulé à la demande de l'utilisateur\n"

#: src/sbackup/core/profile_handler.py:323
msgid "Processing of profile successfully finished (no errors)\n"
msgstr "Traitement du profil terminé avec succès (pas d'erreurs)\n"

#: src/sbackup/core/profile_handler.py:329
#, python-format
msgid "Processing of profile failed with error: %s\n"
msgstr "Le traitement du profil a échoué à cause de l'erreur : %s\n"

#: src/sbackup/core/profile_handler.py:340
#, python-format
msgid "Backup destination: %s"
msgstr "Destination de la sauvegarde : %s"

#: src/sbackup/core/profile_handler.py:344
msgid "Unable to find destination directory."
msgstr "Impossible de trouver le répertoire de destination."

#: src/sbackup/core/profile_handler.py:363
msgid "Unable to find destination directory even after retry."
msgstr ""
"Impossible de trouver le répertoire de destination après une nouvelle "
"tentative."

#: src/sbackup/core/profile_handler.py:364
#: src/sbackup/core/profile_handler.py:371
#, python-format
msgid "Target directory '%(target)s' does not exist."
msgstr "Le répertoire de destination '%(target)s' n'existe pas."

#: src/sbackup/core/profile_handler.py:377
#, python-format
msgid "Unable to access destination: %s"
msgstr "Impossible d'accéder à la destination : %s"

#: src/sbackup/core/snapshot.py:134
msgid "Snapshot is inconsistent: __name is not set "
msgstr "L'instantané est incohérent:__name n'est pas réglé "

#: src/sbackup/core/snapshot.py:145
#, python-format
msgid "Name of snapshot '%s' doesn't match requirement"
msgstr "Le nom de l'instantané '%s' ne correspond pas au schéma attendu."

#: src/sbackup/core/snapshot.py:239
msgid "Snapshot is inconsistent: __snapshotpath is not set "
msgstr "L'instantané est incohérent:__snapshotpath n'est pas réglé "

#: src/sbackup/core/snapshot.py:318
#, python-format
msgid ""
"The snapshot compression format is supposed to be '%s' but the corresponding "
"well named file wasn't found"
msgstr ""
"Le format de compression de l'instantané est censé être '%s' mais le fichier "
"correctement nommé n'a pas été trouvé."

#: src/sbackup/core/snapshot.py:342
#, python-format
msgid ""
"%(file)s doesn't contain valid value. Ignoring incomplete or non - backup "
"directory. "
msgstr ""
"%(file)s ne contient pas de valeur valide. Les dossiers incomplets ou sans "
"rapport avec la sauvegarde seront ignorés. "

#: src/sbackup/core/snapshot.py:478
#, python-format
msgid "Name of Snapshot not valid : %s"
msgstr "Le nom de l'instantané est invalide : %s"

#: src/sbackup/core/snapshot.py:496
#, python-format
msgid "Name of base not valid : %s"
msgstr "Nom du parent invalide : %s"

#: src/sbackup/core/snapshot.py:542
#, python-format
msgid "Invalid name of snapshot: %s"
msgstr "Nom de l'instantané invalide : %s"

#: src/sbackup/core/snapshot.py:546
msgid "Unable to read mandatory file `ver`"
msgstr "Impossible de lire le fichier obligatoire 'ver'"

#: src/sbackup/core/snapshot.py:591
msgid "Base name must be set for incremental backup."
msgstr ""
"Le nom de la base doit être sélectionné pour une sauvegarde incrémentielle."

#: src/sbackup/fs_backend/_fuse_fam.py:106
msgid "Initializing FUSE File Access Manager."
msgstr "Initialisation du gestionnaire d'accès aux fichiers FUSE."

#: src/sbackup/fs_backend/_fuse_fam.py:129
msgid "Terminating FUSE File Access Manager."
msgstr "Clôture du gestionnaire d'accès aux fichiers FUSE."

#: src/sbackup/fs_backend/_fuse_utils.py:45
#: src/sbackup/fs_backend/_gio_utils.py:109
msgid "SFTP"
msgstr "SFTP"

#: src/sbackup/fs_backend/_fuse_utils.py:46
#: src/sbackup/fs_backend/_gio_utils.py:110
msgid "SSH"
msgstr "SSH"

#: src/sbackup/fs_backend/_fuse_utils.py:47
#: src/sbackup/fs_backend/_gio_utils.py:111
msgid "FTP"
msgstr "FTP"

#: src/sbackup/fs_backend/_gio_fam.py:100
msgid "Initializing GIO File Access Manager."
msgstr "Initialisation du gestionnaire d'accès aux fichiers GIO."

#: src/sbackup/fs_backend/_gio_fam.py:114
msgid "Terminating GIO File Access Manager."
msgstr "Clôture du gestionnaire d'accès aux fichiers GIO."

#: src/sbackup/fs_backend/_gio_utils.py:95
msgid "Unknown error code:"
msgstr ""

#: src/sbackup/fs_backend/_gio_utils.py:112
msgid "NFS"
msgstr "NFS"

#: src/sbackup/fs_backend/_gio_utils.py:113
msgid "SMB"
msgstr "SMB"

#: src/sbackup/fs_backend/_gio_utils.py:193
msgid "Unable to get mount: path is not mounted"
msgstr "Impossible d'obtenir le montage: le chemin n'est pas monté."

#: src/sbackup/fs_backend/_gio_utils.py:195
msgid "Unable to get mount: path not found when mounting (is probably local)"
msgstr ""
"Impossible d'obtenir le montage: aucun chemin trouvé pendant le montage (est "
"probablement local)"

#: src/sbackup/fs_backend/_gio_utils.py:197
#: src/sbackup/fs_backend/_gio_utils.py:200
#, python-format
msgid "Unable to get mount: %s"
msgstr "Impossible de de monter: %s"

#: src/sbackup/fs_backend/_gio_utils.py:250
msgid "Unable to un-set mount flag: path is not stored"
msgstr ""
"Impossible de désactiver le drapeau de montage: le chemin n'est pas conservé."

#: src/sbackup/fs_backend/_gio_utils.py:352
msgid "Path is already mounted."
msgstr "Le chemin est déjà monté."

#: src/sbackup/fs_backend/_gio_utils.py:380
msgid "Path is not mounted."
msgstr "Le chemin n'est pas monté."

#: src/sbackup/fs_backend/_gio_utils.py:887
msgid "test specified path for existence using GIO"
msgstr "Vérification du chemin spécifié utilisant GIO"

#: src/sbackup/fs_backend/_gio_utils.py:900
msgid "Create testdir"
msgstr "Créer un répertoire de test"

#: src/sbackup/fs_backend/_gio_utils.py:905
msgid "Test testfile for existence"
msgstr "Vérification de l'existence de testfile"

#: src/sbackup/fs_backend/_gio_utils.py:912
msgid "Create file"
msgstr "Créer un fichier"

#: src/sbackup/fs_backend/_gio_utils.py:919
msgid "Re-read test file"
msgstr "Relecture du fichier test"

#: src/sbackup/fs_backend/_gio_utils.py:931
msgid "Remove file"
msgstr "Supprimer le fichier"

#: src/sbackup/fs_backend/_gio_utils.py:933
msgid "Remove dir"
msgstr "Supprimer le dossier"

#: src/sbackup/fuse_plugins/__init__.py:160
#, python-format
msgid "Unable to import plugin `%(plugin_name)s`: %(error_cause)s "
msgstr ""
"Impossible de d'importer le module externe `%(plugin_name)s`: "
"%(error_cause)s "

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:117
#, python-format
msgid ""
"Couldn't found external application 'curlftpfs' needed for handling of ftp "
"sites: %s"
msgstr ""
"Impossible de trouver l'application externe 'curlftpfs' nécessaire à la "
"gestion des destinations FTP: %s"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:127
#: src/sbackup/fuse_plugins/sftpFuseFAM.py:124
#, python-format
msgid "Couldn't mount '%(server)s' into '%(mountpoint)s' : %(error)s"
msgstr "Impossible de monter %(server)s' sur '%(mountpoint)s' : %(error)s"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:141
msgid "FTP schema is: ftp://user:pass@server/anything"
msgstr "Le schéma FTP est : ftp://utilisateur:motdepasse@serveur/repertoire"

#: src/sbackup/fuse_plugins/ftpFuseFAM.py:188
#, python-format
msgid ""
"Error matching the schema 'ftp://user:pass@server/anything' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Problème sur la correspondance du schéma 'ftp://user:pass@server/anything' "
"avec '%s' (le '/' après 'server' est obligatoire)"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:114
#, python-format
msgid ""
"Couldn't found external application 'curlftpfs' needed for handling of sftp "
"sites: %s"
msgstr ""
"Impossible de trouver l'application externe 'curlftpfs' nécessaire à la "
"gestion des sites sftp : %s"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:138
msgid "SFTP schema is like : sftp://user:pass@server/anything"
msgstr ""
"Le modèle SFTP est : sftp://utilisateur:mot_de_passe@serveur/suite_du_chemin"

#: src/sbackup/fuse_plugins/sftpFuseFAM.py:185
#, python-format
msgid ""
"Error matching the schema 'sftp://user:pass@server/anything' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Problème sur la correspondance du schéma 'sftp://user:pass@server/anything' "
"avec '%s' (le '/' après 'server' est obligatoire)"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:81
#, python-format
msgid ""
"Error matching the schema 'ssh://user:pass@example.com/home/' with '%s' (The "
"'/' after server is mandatory)"
msgstr ""
"Problème sur la correspondance du schéma 'ssh://user:pass@example.com/home/' "
"avec '%s' (le '/' après 'server' est obligatoire)"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:137
#, python-format
msgid ""
"The sshfs command '%(command)s' didn't perform normally. Output => "
"%(erroroutput)s "
msgstr ""
"La commande sshfs '%(command)s' ne s'est pas exécutée normalement. Résultat "
"=> %(erroroutput)s "

#: src/sbackup/fuse_plugins/sshFuseFAM.py:142
msgid "SSH schema is like : ssh://user:pass@example.com:33/home/user/backup/"
msgstr ""
"Le schéma SSH est le suivant : "
"ssh://utilisateur:motdepasse@serveur.com:33/home/user/backup/"

#: src/sbackup/fuse_plugins/sshFuseFAM.py:158
#, python-format
msgid ""
"Error matching the schema 'ssh://user:pass@example.com:21/home/' with '%s' "
"(The '/' after server is mandatory)"
msgstr ""
"Impossible de faire correspondre le schéma "
"'ssh://utilisateur:motdepasse@serveur.com:21/home/' avec '%s' (Le '/' après "
"le nom du serveur est obligatoire)"

#: src/sbackup/ui/GladeWindow.py:210
msgid ""
"Please make sure the missing directory exists (e.g. by mounting an external "
"disk) or change the specified target."
msgstr ""
"S.V.P., assurez-vous que le répertoire manquant existe (en montant un disque "
"externe par exemple) ou changez la cible spécifiée."

#: src/sbackup/ui/GladeWindow.py:211
#, python-format
msgid ""
"While accessing the backup destination following error occurred:\n"
"\n"
"%s."
msgstr ""
"En accédant à la destination de la sauvegarde, l'erreur suivante s'est "
"produite.\n"
"\n"
"%s."

#: src/sbackup/ui/GladeWindow.py:212 src/sbackup/ui/restoregui.py:797
msgid "Simple Backup error"
msgstr "Erreur Simple Backup"

#: src/sbackup/ui/GladeWindow.py:213
msgid "Unable to access backup destination"
msgstr "Impossible d'accéder à la destination de sauvegarde"

#: src/sbackup/ui/configgui.py:125
msgid "Scheduled backups are available for Administrator users only."
msgstr ""
"Les sauvegardes programmées sont disponibles seulement pour les "
"administrateurs."

#: src/sbackup/ui/configgui.py:135 src/sbackup/ui/configgui.py:144
msgid "Name"
msgstr "Nom"

#: src/sbackup/ui/configgui.py:152
msgid "File Type"
msgstr "Type de Fichier"

#: src/sbackup/ui/configgui.py:187
msgid "Enable"
msgstr "Activé"

#: src/sbackup/ui/configgui.py:188
msgid "Profile Name"
msgstr "Nom du profil"

#: src/sbackup/ui/configgui.py:285
#, python-format
msgid ""
"Invalid or empty regular expression ('%s') found in configuration. Removed."
msgstr ""
"Une expression régulière invalide ou vide ('%s') a été trouvée dans la "
"configuration et supprimée."

#: src/sbackup/ui/configgui.py:292
#, python-format
msgid ""
"Invalid or empty regular expressions found\n"
"in configuration file:\n"
"'%s'\n"
"\n"
"These expressions are not used and were\n"
"removed from the configuration."
msgstr ""
"Des expressions régulières invalides ou vides ont été trouvées\n"
"dans le fichier de configuration :\n"
"'%s'\n"
"\n"
"Ces expressions ne sont pas utilisées et ont été\n"
"supprimées de la configuration."

#: src/sbackup/ui/configgui.py:548
msgid ""
"Please make sure the missing directory exists (e.g. by mounting an external "
"disk) or change the specified target to an existing one."
msgstr ""
"Assurez vous s'il vous plait que le répertoire manquant existe (par exemple "
"en montant un disque externe) ou en changer le répertoire de destination à "
"un répertoire existant."

#: src/sbackup/ui/configgui.py:549
#, python-format
msgid ""
"Backup target `%s` does not exist.\n"
"\n"
"Attention: The target will be set to the default value. Check this on the "
"destination settings page before saving the configuration."
msgstr ""
"La cible de la sauvegarde '%s' n'existe pas.\n"
"\n"
"Attention: La cible utilisée sera celle par défaut. Vérifier ceci dans la "
"page des réglages de la destination avant de sauvegarder la configuration."

#: src/sbackup/ui/configgui.py:551
msgid "Unable to open backup target"
msgstr "Impossible d'ouvrir la destination de sauvegarde."

#: src/sbackup/ui/configgui.py:623
#, python-format
msgid ""
"%s\n"
"\n"
"A backup profile using default values was created. Save the new "
"configuration in order to use it or check your existing configuration file "
"manually.\n"
"\n"
"Important note: Saving will overwrite your existing invalid configuration."
msgstr ""
"%s\n"
"\n"
"Un profil de sauvegarde utilisant les valeurs par défaut a été créé. "
"Enregistrez la nouvelle configuration afin de l'utiliser ou vérifiez "
"manuellement votre fichier de configuration existant.\n"
"\n"
"Important : Sauvegarder va écraser la configuration invalide existante."

#: src/sbackup/ui/configgui.py:641
#, python-format
msgid "Current profile: %s"
msgstr "Profil actuel : %s"

#: src/sbackup/ui/configgui.py:732
msgid "Already included item !"
msgstr "Élément déjà ajouté !"

#: src/sbackup/ui/configgui.py:747
msgid "Already excluded item !"
msgstr "Élément déjà enlevé !"

#: src/sbackup/ui/configgui.py:759 src/sbackup/ui/configgui.py:1458
msgid "Empty expression. Please enter a valid regular expression."
msgstr "Expression vide. Veuillez entrer une expression régulière valide."

#: src/sbackup/ui/configgui.py:774 src/sbackup/ui/configgui.py:1472
msgid "Provided regular expression is not valid."
msgstr "L'expression régulière fournie est invalide."

#: src/sbackup/ui/configgui.py:782
msgid "Empty filename or path. Please enter a valid filename or path."
msgstr ""
"Nom de fichier ou chemin vide. Veuillez entrer un nom de fichier ou un "
"chemin valide."

#: src/sbackup/ui/configgui.py:791
msgid ""
"It seems the path you entered does not exists. Do you want to add this "
"incorrect path?"
msgstr ""
"Le chemin que vous avez entré ne semble pas exister. Voulez-vous tout de "
"même le conserver ?"

#: src/sbackup/ui/configgui.py:820
msgid "Save configuration as..."
msgstr "Sauver la configuration en tant que ..."

#: src/sbackup/ui/configgui.py:887
msgid ""
"A backup process is now executed in the background.\n"
"\n"
"You can monitor the progress of the backup by means of the status indicator "
"displayed in the notification area."
msgstr ""
"Un processus de sauvegarde s'effectue actuellement en arrière-plan.\n"
"\n"
"Vous pouvez surveiller l'avancement de la sauvegarde au moyen de "
"l'indicateur d'état situé dans la zone de notification."

#: src/sbackup/ui/configgui.py:888
msgid "Backup process started"
msgstr "Processus de sauvegarde démarré"

#: src/sbackup/ui/configgui.py:893 src/sbackup/ui/configgui.py:1661
#: src/sbackup/ui/configgui.py:2038
#, python-format
msgid "Unexpected dialog response: %s"
msgstr "Réponse de dialogue inattendue: %s"

#: src/sbackup/ui/configgui.py:967
msgid "Include file..."
msgstr "Inclure le fichier ..."

#: src/sbackup/ui/configgui.py:974 src/sbackup/ui/configgui.py:1029
msgid "All files"
msgstr "Tous fichiers"

#: src/sbackup/ui/configgui.py:994
msgid "Include folder..."
msgstr "Inclure le répertoire ..."

#: src/sbackup/ui/configgui.py:1022
msgid "Exclude file..."
msgstr "Exclure le fichier ..."

#: src/sbackup/ui/configgui.py:1045
msgid "Exclude folder..."
msgstr "Exclure le répertoire ..."

#: src/sbackup/ui/configgui.py:1310
msgid "Configuration has changed"
msgstr "La configuration a été modifiée."

#: src/sbackup/ui/configgui.py:1311
msgid ""
"There are unsaved modifications. Please save the configuration or revert "
"these changes before testing the mail settings."
msgstr ""
"Il y a des modifications non sauvegardées. Merci de sauver la configuration "
"ou de réinitialiser ces changements avant de tester les réglages courriel."

#: src/sbackup/ui/configgui.py:1317 src/sbackup/ui/configgui.py:1323
msgid "Test mail settings"
msgstr "Tester les réglages courriel"

#: src/sbackup/ui/configgui.py:1318
msgid "The test failed with following output:"
msgstr "Le test a échoué avec le résultat suivant :"

#: src/sbackup/ui/configgui.py:1324
msgid "The test was successful."
msgstr "Le test est réussi."

#: src/sbackup/ui/configgui.py:1407 src/sbackup/ui/configgui.py:1450
#, python-format
msgid ""
"The given expression contains unsupported characters ('%s'). Currently it is "
"not possible to use these characters in exclude expressions."
msgstr ""
"L'expression fournie contient des caractères non supportés ('%s'). Il n'est "
"actuellement pas possible d'utiliser ces caractères dans les expressions "
"d'exclusion."

#: src/sbackup/ui/configgui.py:1410 src/sbackup/ui/configgui.py:1453
msgid "Unsupported character"
msgstr "Caractère non supporté."

#: src/sbackup/ui/configgui.py:1621
msgid "Connect to remote destination"
msgstr "Connexion à la destination distante"

#: src/sbackup/ui/configgui.py:1669
msgid "Mount was sucessful (no errors)"
msgstr "Le montage a réussi (pas d'erreurs)"

#: src/sbackup/ui/configgui.py:1680
msgid "Unable to mount host"
msgstr "Impossible de monter l'hôte"

#: src/sbackup/ui/configgui.py:1689
msgid "Umount was sucessful (no errors)"
msgstr "La démontage a réussi (pas d'erreurs)"

#: src/sbackup/ui/configgui.py:1699
msgid "Unable to unmount host"
msgstr "Impossible de démonter l'hôte"

#: src/sbackup/ui/configgui.py:1729
msgid "Perfom tests on remote host"
msgstr "Tests de performances sur l'hôte distant"

#: src/sbackup/ui/configgui.py:1742
msgid "Unable to access remote destination"
msgstr "Impossible d'accéder à la destination distante"

#: src/sbackup/ui/configgui.py:1746
msgid "All tests passed"
msgstr "Tous les tests ont réussi"

#: src/sbackup/ui/configgui.py:1910
msgid ""
"The given name of the new profile is empty. Please enter a valid profile "
"name."
msgstr ""
"Le nom fourni pour le nouveau profil est vide. Merci d'entrer un nom de "
"profil valide."

#: src/sbackup/ui/configgui.py:1911 src/sbackup/ui/configgui.py:1918
msgid "Profile name not valid"
msgstr "Le nom de profil n'est pas valide."

#: src/sbackup/ui/configgui.py:1917
msgid ""
"The given name of the new profile already exists. Please enter another name."
msgstr ""
"Le nom donné au nouveau profil existe déjà. Merci d'entrer un autre nom."

#: src/sbackup/ui/configgui.py:1919
msgid "Renaming of profiles is not supported."
msgstr "Le changement de nom des profils n'est pas pris en charge."

#: src/sbackup/ui/configgui.py:1949
#, python-format
msgid ""
"<b>Delete configuration profile?</b>\n"
"\n"
"You are trying to remove a configuration profile. You will not be able to "
"restore it. If you are not sure, use the 'enable|disable' functionality "
"instead.\n"
"\n"
"Do you really want to delete the profile '%(name)s'?"
msgstr ""
"<b>Effacer le profil de configuration ?</b>\n"
"\n"
"Vous essayez de supprimer un profil de configuration. Vous ne serez pas en "
"mesure de le restaurer. Si vous n'êtes pas sûr, utilisez plutôt la "
"fonctionnalité 'activer|désactiver'.\n"
"\n"
"Voulez-vous vraiment supprimer le profil '%(name)s' ?"

#: src/sbackup/ui/configgui.py:2022
#, python-format
msgid ""
"<big><b>Set default values for current profile?</b></big>\n"
"This will restore the default values for the profile currently edited: "
"'%s'.\n"
"\n"
"These predefined settings are recommended for most users. Check whether they "
"are appropriate for your use before saving the changed configuration."
msgstr ""
"<big><b>Activer les réglages par défaut pour le profil courant ?</b></big>\n"
"Cette opération va restaurer les réglages par défaut pour le profil en cours "
"d'édition: '%s'.\n"
"\n"
"Ces réglages par défaut sont recommandés pour la plupart des utilisateurs. "
"Merci de vérifier s'ils conviennent à votre usage avant de sauvegarder la "
"configuration."

#: src/sbackup/ui/configgui.py:2032
msgid "Default settings are being applied."
msgstr "Les réglages par défaut sont appliqués."

#: src/sbackup/ui/configgui.py:2057
msgid ""
"<b>Unable to remove default profile</b>\n"
"\n"
"The default profile cannot be removed. In the case you want to use just a "
"single profile, please set up the default profile accordingly."
msgstr ""
"<b>Impossible de supprimer le profil par défaut</b>\n"
"\n"
"Le profil par défaut ne peut être enlevé. Dans le cas où vous voulez "
"utiliser seulement un seul profil, s.v.p. mettre en place le profil par "
"défaut en conséquence."

#: src/sbackup/ui/configgui.py:2071
msgid ""
"<b>Unable to remove default profile</b>\n"
"\n"
"The default profile cannot be disabled. In the case you want to use just a "
"single profile, please set up the default profile accordingly."
msgstr ""
"<b>Impossible de supprimer le profil par défaut</b>\n"
"\n"
"Le profil par défaut ne peut être désactivé. Dans le cas où vous voulez "
"utiliser seulement un seul profil, s.v.p. mettre en place le profil par "
"défaut en conséquence."

#: src/sbackup/ui/configgui.py:2085
msgid ""
"<b>No backup profile found.</b>\n"
"\n"
"No default profile was found. You are probably running Simple Backup for the "
"first time. A backup profile using default values was created.\n"
"\n"
"Please modify the settings according to your needs and save the "
"configuration in order to use it."
msgstr ""
"<b>Aucun profil de sauvegarde trouvé.</b>\n"
"\n"
"Pas de profil par défaut a été constaté. Vous utilisez probablement Simple "
"Backup pour la première fois. Un profil de sauvegarde utilisant les valeurs "
"par défaut a été créé.\n"
"\n"
"S.V.P., modifier les paramètres selon vos besoins et enregistrer la "
"configuration afin de l'utiliser."

#: src/sbackup/ui/configgui.py:2098
msgid "Please select a profile."
msgstr "Veuillez sélectionner un profil."

#: src/sbackup/ui/indicator_core.py:117
msgid "there was a problem initializing the pynotify module"
msgstr "Échec de l'initialisation du module de notification"

#: src/sbackup/ui/indicator_core.py:148 src/sbackup/ui/indicator_core.py:197
#: src/sbackup/ui/indicator_core.py:225 src/sbackup/ui/indicator_core.py:247
#, python-format
msgid "Connection to notification-daemon failed: %s"
msgstr "La connexion au démon de notification a échoué : %s"

#: src/sbackup/ui/indicator_core.py:310
msgid "Simple Backup"
msgstr "Simple Backup"

#: src/sbackup/ui/indicator_core.py:316
msgid "Show message windows"
msgstr "Montrer les fenêtres de notification"

#: src/sbackup/ui/indicator_core.py:358 src/sbackup/ui/indicator_core.py:816
msgid "Cancel Backup"
msgstr "Annuler la Sauvegarde"

#: src/sbackup/ui/indicator_core.py:420
msgid "Connection to D-Bus service lost."
msgstr "Connection to D-Bus service lost."

#: src/sbackup/ui/indicator_core.py:445
msgid "re-connection was successful"
msgstr "La reconnexion a réussi"

#: src/sbackup/ui/indicator_core.py:484
msgid ""
"No backup in progress and connection to D-Bus service lost. Simple Backup "
"Indicator is being terminated."
msgstr ""
"Pas de sauvegarde en cours et la connexion au service D-Bus est perdu. "
"L'indicateur Simple Backup est terminé."

#: src/sbackup/ui/indicator_core.py:513
msgid "Starting backup session"
msgstr "Démarrage d'une session de sauvegarde"

#: src/sbackup/ui/indicator_core.py:527
msgid "Ending Backup Session"
msgstr "Fin de la session de sauvegarde"

#: src/sbackup/ui/indicator_core.py:540
#, python-format
msgid "Unknown D-Bus event `%s` received."
msgstr "Réception de l'événement D-Bus '%s' reçu."

#: src/sbackup/ui/indicator_core.py:581
msgid "Try again"
msgstr "Réessayer"

#: src/sbackup/ui/indicator_core.py:651
msgid ""
"Attempt of starting another instance of Simple Backup while this one is "
"already running."
msgstr ""
"Tentative de démarrage d'une autre instance de Simple Backup alors que celui-"
"ci est déjà en cours d'exécution."

#: src/sbackup/ui/indicator_core.py:661
#, python-format
msgid "Unable to present window: %s"
msgstr "Impossible d'afficher la fenêtre:%s"

#: src/sbackup/ui/indicator_core.py:677
#, python-format
msgid "Do you really want to cancel the backup process of profile '%s'?"
msgstr ""
"Êtes-vous certain de vouloir annuler le processus de sauvegarde du profil "
"'%s'?"

#: src/sbackup/ui/indicator_core.py:681
msgid "Cancel running backup process?"
msgstr "Annuler le processus de sauvegarde en cours d'exécution?"

#: src/sbackup/ui/indicator_core.py:698
msgid "Unable to cancel backup process."
msgstr "Impossible d'annuler le processus de sauvegarde."

#: src/sbackup/ui/indicator_core.py:760
#, python-format
msgid "Profile: %s"
msgstr "Profil : %s"

#: src/sbackup/ui/indicator_core.py:761
#, python-format
msgid "Size of backup: %s"
msgstr "Taille de la sauvegarde: %s"

#: src/sbackup/ui/indicator_core.py:762
#, python-format
msgid "%.1f%% processed"
msgstr "%.1f%% effectué"

#: src/sbackup/ui/indicator_core.py:763
#, python-format
msgid "Remaining time: %s"
msgstr "Temps restant : %s"

#: src/sbackup/ui/indicator_core.py:809
#, python-format
msgid "PID of backup process being canceled: %s"
msgstr "Le PID du processus de sauvegarde est annulé: %s"

#: src/sbackup/ui/indicator_core.py:814
#, python-format
msgid "Unable to send signal to process %s"
msgstr "impossible d'envoyer un signal au processus: %s"

#: src/sbackup/ui/indicator_core.py:815
#, python-format
msgid "Sending signal using gksu to process %s"
msgstr "Envoie d'un signal en utilisant gksu au processus %s"

#: src/sbackup/ui/indicator_core.py:819
#, python-format
msgid "Sending signal using gksu to process %s failed"
msgstr "L'envoie d'un signal en utilisant gksu au processus %s a échoué"

#: src/sbackup/ui/indicator_core.py:854
#, python-format
msgid ""
"<b>Unable to find specified target directory</b>\n"
"\n"
"The specified target directory '%s' was not found.\n"
"\n"
msgstr ""
"<b>Impossible de trouver le répertoire cible indiqué</b>\n"
"\n"
"Le répertoire cible indiqué '%s' n'existe pas.\n"
"\n"

#: src/sbackup/ui/indicator_core.py:855
#, python-format
msgid ""
"You can try to use the specified target again or cancel the profile "
"execution. The specified destination is automatically used in %s seconds."
msgstr ""
"Vous pouvez essayer d'utiliser à nouveau la cible spécifiée ou annuler "
"l'exécution du profil. La destination spécifiée sera automatiquement "
"utilisée dans %s secondes."

#: src/sbackup/ui/indicator_core.py:882 src/sbackup/ui/indicator_core.py:924
#: src/sbackup/ui/indicator_core.py:942 src/sbackup/ui/indicator_core.py:950
#: src/sbackup/ui/indicator_core.py:967
msgid "unknown"
msgstr "inconnu"

#: src/sbackup/ui/indicator_core.py:903
msgid "less than 1 minute"
msgstr "Il y a moins d'une minute"

#: src/sbackup/ui/indicator_core.py:905
msgid "about 2 minutes"
msgstr "Il y a 2 minutes"

#: src/sbackup/ui/indicator_core.py:907
msgid "about 3 minutes"
msgstr "Il y a environ 3 minutes"

#: src/sbackup/ui/indicator_core.py:909
msgid "about 4 minutes"
msgstr "Il y a environ 4 minutes"

#: src/sbackup/ui/indicator_core.py:911
msgid "about 5 minutes"
msgstr "Il y a environ 5 minutes"

#: src/sbackup/ui/indicator_core.py:913
#, python-format
msgid "about %.0f minutes"
msgstr "Il y a environ %.0f minutes"

#: src/sbackup/ui/indicator_core.py:923
msgid "In progress"
msgstr "En cours"

#: src/sbackup/ui/indicator_core.py:933 src/sbackup/ui/indicator_core.py:959
msgid "finished"
msgstr "Terminé"

#: src/sbackup/ui/indicator_core.py:941
msgid "Progress: unknown"
msgstr "Progression : inconnue"

#: src/sbackup/ui/indicator_core.py:948
msgid "Backup is being prepared"
msgstr "La sauvegarde est en préparation"

#: src/sbackup/ui/indicator_core.py:949
msgid "Collecting file info"
msgstr "Collecte des informations sur les fichiers"

#: src/sbackup/ui/indicator_core.py:958
msgid "Progress: canceled"
msgstr "Progression : annulée"

#: src/sbackup/ui/indicator_core.py:980
msgid "An error occurred"
msgstr "Une erreur est survenue"

#: src/sbackup/ui/indicator_core.py:1013
msgid "Another `Simple Backup Indicator` is already running."
msgstr "Un autre 'Indicateur Simple Backup' est déjà en cours d'exécution."

#: src/sbackup/ui/indicator_core.py:1037
#, python-format
msgid "Initialization of GTK+ failed: %s"
msgstr "L'initialisation de GTK+ a échoué : %s"

#: src/sbackup/ui/misc.py:294
#, python-format
msgid "%s   (Administrator mode)"
msgstr "%s   (mode administrateur)"

#: src/sbackup/ui/misc.py:324
#, python-format
msgid ""
"An uncaught error occurred. Close this message window and restart the "
"application.\n"
"\n"
"Please report this error on https://bugs.launchpad.net/sbackup.\n"
"\n"
"%s"
msgstr ""
"Une erreur non interceptée s'est produite. Fermer cette fenêtre et "
"redémarrer l'application.\n"
"\n"
"S'il vous plaît, signalez cette erreur sur "
"https://bugs.launchpad.net/sbackup.\n"
"\n"
"%s"

#: src/sbackup/ui/misc.py:325
msgid "Sorry, this should not have happened"
msgstr "Désolé, cela n'aurait pas du se produire"

#: src/sbackup/ui/restoregui.py:78
msgid "Restore..."
msgstr "Restoration ..."

#: src/sbackup/ui/restoregui.py:79 data/ui/sbackup-restore.glade:765
msgid "Restore as..."
msgstr "Restoration en tant que ..."

#: src/sbackup/ui/restoregui.py:80
msgid "Revert..."
msgstr "Rétrograder ..."

#: src/sbackup/ui/restoregui.py:81 data/ui/sbackup-restore.glade:797
msgid "Revert as..."
msgstr "Rétrograder comme ..."

#: src/sbackup/ui/restoregui.py:128 data/ui/sbackup-restore.glade:111
#: data/ui/sbackup-restore.glade:511
msgid "n.a."
msgstr "n.d."

#: src/sbackup/ui/restoregui.py:146 src/sbackup/ui/restoregui.py:163
msgid "Snapshots"
msgstr "Instantanés"

#: src/sbackup/ui/restoregui.py:147
msgid "Version"
msgstr "Version"

#: src/sbackup/ui/restoregui.py:154 data/ui/sbackup-config.glade:4063
msgid "Path"
msgstr "Chemin"

#: src/sbackup/ui/restoregui.py:159
msgid "State"
msgstr "État"

#: src/sbackup/ui/restoregui.py:187
msgid "Connect to destination..."
msgstr "Connexion à la destination..."

#: src/sbackup/ui/restoregui.py:211
msgid "Specified path does not exist"
msgstr "Les chemins spécifiés n'existent pas"

#: src/sbackup/ui/restoregui.py:257
msgid "Ready"
msgstr "Prêt"

#: src/sbackup/ui/restoregui.py:371
msgid "No destination specified"
msgstr "Aucune destination spécifiée"

#: src/sbackup/ui/restoregui.py:388
msgid "Choose a source folder"
msgstr "Choisissez un dossier source"

#: src/sbackup/ui/restoregui.py:405
msgid "Reading snapshots..."
msgstr "Lecture des instantanés..."

#: src/sbackup/ui/restoregui.py:428
msgid "Reading directory content..."
msgstr "Lecture du contenu du répertoire..."

#: src/sbackup/ui/restoregui.py:458 src/sbackup/ui/restoregui.py:496
msgid "Loading ..."
msgstr "Chargement en cours..."

#: src/sbackup/ui/restoregui.py:539
msgid "Reading files from snapshot..."
msgstr "Lecture des fichiers de l'instantané..."

#: src/sbackup/ui/restoregui.py:552
msgid "Simple Backup restore error"
msgstr "Erreur de restauration Simple Backup"

#: src/sbackup/ui/restoregui.py:553
msgid "An error occurred while reading snapshot"
msgstr "Une erreur s'est produite pendant la lecture de l'instantané."

#: src/sbackup/ui/restoregui.py:557
msgid "This snapshot seems empty."
msgstr "Cet instantané semble vide."

#: src/sbackup/ui/restoregui.py:576
msgid "Reading backup snapshot..."
msgstr "Lecture de la sauvegarde ..."

#: src/sbackup/ui/restoregui.py:599
msgid "No backups found for this day."
msgstr "Pas de sauvegarde trouvée pour ce jour."

#: src/sbackup/ui/restoregui.py:614 src/sbackup/ui/restoregui.py:771
#, python-format
msgid ""
"Snapshot version is not supported (Only %(supportedversion)s is supported). "
"Version '%(currentversion)s' found."
msgstr ""

#: src/sbackup/ui/restoregui.py:643
msgid "Select restore location"
msgstr "Sélectionnez l'emplacement de restauration"

#: src/sbackup/ui/restoregui.py:662
#, python-format
msgid "Do you really want to revert '%s'?"
msgstr "Voulez-vous vraiment rétrograder '%s'?"

#: src/sbackup/ui/restoregui.py:676
msgid "Select revert location"
msgstr "Choisissez la destination de la rétrogradation."

#: src/sbackup/ui/restoregui.py:685
#, python-format
msgid "Do you really want to revert '%(source)s' to '%(dir)s'?"
msgstr "Voulez-vous vraiment rétrograder '%(source)s' vers '%(dir)s'?"

#: src/sbackup/ui/restoregui.py:763
msgid "Reading snapshot..."
msgstr "Lecture de l'instantané..."

#: src/sbackup/ui/restoregui.py:782
#, python-format
msgid ""
"Deleting the selected snapshot `%s` will remove your backup for this "
"particular point in time. You cannot undo this operation.\n"
"\n"
"Are you sure that you want to remove the snapshot permanently?"
msgstr ""
"La suppression de l'instantané sélectionné '%s' supprimera votre sauvegarde "
"à ce point précis dans le temps. Cette opération est irréversible.\n"
"\n"
"Êtes-vous certain que vous souhaitez supprimer l'instantané?"

#: src/sbackup/ui/restoregui.py:786
msgid "Do you really want to remove snapshot?"
msgstr "Voulez-vous vraiment supprimer l'instantané ?"

#: src/sbackup/ui/restoregui.py:795
#, python-format
msgid ""
"While attempting to delete snapshot the following error occurred:\n"
"%s"
msgstr ""
"Lors de la suppression de l'instantané, l'erreur suivante est arrivée :\n"
"%s"

#: src/sbackup/ui/restoregui.py:798
msgid "Unable to delete snapshot"
msgstr "Impossible d'effacer l'instantané"

#: src/sbackup/ui/restoregui.py:804
#, python-format
msgid ""
"The selected snapshot '%s' is referenced by more recent snapshots as base "
"snapshot.\n"
"\n"
"Remove all of these child snapshots in order to delete this snapshot."
msgstr ""
"L'instantané sélectionné '%s' est référencé par des instantanés plus récents "
"comme instantané de base.\n"
"\n"
"Effacer tous ces instantanés dépendants pour supprimer cet instantané."

#: src/sbackup/ui/restoregui.py:806
msgid "Unable to remove snapshot"
msgstr "Impossible de supprimer l'instantané"

#: src/sbackup/ui/restoregui.py:814
msgid "Termination in progress"
msgstr "Annulation en cours"

#: src/sbackup/ui/restoregui.py:830 src/sbackup/ui/restoregui.py:836
#: src/sbackup/ui/restoregui.py:842 src/sbackup/ui/restoregui.py:848
#: data/ui/sbackup-restore.glade:959
msgid "Simple Backup restoration"
msgstr "Restauration de Simple Backup"

#: src/sbackup/ui/restoregui.py:831 src/sbackup/ui/restoregui.py:837
msgid "<b>Restoring of selected files</b>"
msgstr "Restauration des fichiers sélectionnés"

#: src/sbackup/ui/restoregui.py:832
#, python-format
msgid "Restoring of <tt>'%s'</tt> is in progress."
msgstr "Restauration de <tt>'%s'</tt> en cours"

#: src/sbackup/ui/restoregui.py:833
#, python-format
msgid "Restoring of <tt>'%s'</tt> was successful."
msgstr "Restauration de <tt>'%s'</tt> réussi"

#: src/sbackup/ui/restoregui.py:834
#, python-format
msgid ""
"Restoring of <tt>'%s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""
"La restauration de <tt>'%s'</tt> n'a pas abouti.\n"
"\n"
"L'erreur suivante s'est produite:\n"

#: src/sbackup/ui/restoregui.py:838
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> is in progress."
msgstr ""
"Restauration de <tt>'%(source)s'</tt>\n"
"dans <tt>'%(dirname)s'</tt> en cours."

#: src/sbackup/ui/restoregui.py:839
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was successful."
msgstr ""
"Restauration de <tt>'%(source)s'</tt>\n"
"dans <tt>'%(dirname)s'</tt> a réussi."

#: src/sbackup/ui/restoregui.py:840
#, python-format
msgid ""
"Restoring of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""
"La restauration de <tt>'%(source)s'</tt>\n"
"vers <tt>'%(dirname)s'</tt> n'a pas abouti.\n"
"\n"
"L'erreur suivante s'est produite:\n"

#: src/sbackup/ui/restoregui.py:843 src/sbackup/ui/restoregui.py:849
msgid "<b>Reverting selected files</b>"
msgstr "Rétrograder les fichiers sélectionnés"

#: src/sbackup/ui/restoregui.py:844
#, python-format
msgid "Reverting of <tt>'%s'</tt> is in progress.\n"
msgstr "Rétrogradation de <tt>'%s'</tt> en cours.\n"

#: src/sbackup/ui/restoregui.py:845
#, python-format
msgid "Reverting of <tt>'%s'</tt> was successful."
msgstr "La rétrogradation de <tt>'%s'</tt> a réussi."

#: src/sbackup/ui/restoregui.py:846
#, python-format
msgid ""
"Reverting of <tt>'%s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""
"La rétrogradation de <tt>'%s'</tt> n'a pas abouti\n"
"\n"
"L'erreur suivante s'est produite:\n"

#: src/sbackup/ui/restoregui.py:850
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> is in progress."
msgstr ""
"Rétrogradation de <tt>'%(source)s'</tt> \n"
"dans <tt>'%(dirname)s'</tt> en cours."

#: src/sbackup/ui/restoregui.py:851
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was successful."
msgstr ""
"La rétrogradation de <tt>'%(source)s'</tt>\n"
"vers<tt>'%(dirname)s'</tt> a réussi."

#: src/sbackup/ui/restoregui.py:852
#, python-format
msgid ""
"Reverting of <tt>'%(source)s'</tt>\n"
"to <tt>'%(dirname)s'</tt> was not successful.\n"
"\n"
"Following error occurred:\n"
msgstr ""
"La rétrogradation de <tt>'%(source)s'</tt>\n"
"vers <tt>'%(dirname)s'</tt> n'a pas abouti\n"
"\n"
"L'erreur suivante s'est produite:\n"

#: src/sbackup/util/__init__.py:439
#, python-format
msgid "%(mb)d MiB %(kb)d KiB"
msgstr "%(mb)d Mo %(kb)d Ko"

#: src/sbackup/util/__init__.py:442
#, python-format
msgid "%(mb)d MiB %(kb)d KiB %(bytes)d"
msgstr "%(mb)d Mo %(kb)d Ko %(bytes)d"

#: src/sbackup/util/__init__.py:447
#, python-format
msgid "%(mb)d MB %(kb)d kB"
msgstr "%(mb)d MB %(kb)d kB"

#: src/sbackup/util/__init__.py:450
#, python-format
msgid "%(mb)d MB %(kb)d kB %(bytes)d"
msgstr "%(mb)d MB %(kb)d kB %(bytes)d"

#: src/sbackup/util/filecollect.py:335
#, python-format
msgid "File '%(file)s' is not accessable with error '%(error)s'."
msgstr "Le fichier '%(file)s' n'est pas accessible, erreur '%(error)s'"

#: src/sbackup/util/filecollect.py:351
#, python-format
msgid ""
"File '%(file)s' cannot be opened for read access. Operation timed out."
msgstr ""
"Impossible d'ouvrir le fichier '%(file)s' en lecture. Délai dépassé pour "
"l'opération."

#: src/sbackup/util/filecollect.py:355
#, python-format
msgid ""
"File '%(file)s' cannot be opened for read access with error '%(error)s'."
msgstr ""
"Le fichier '%(file)s' ne peut être ouvert en lecture, erreur '%(error)s'"

#: src/sbackup/util/filecollect.py:372
#, python-format
msgid "Symbolic link '%(path)s' -> '%(ln_target)s' is a circular symlink."
msgstr ""
"Le lien symbolique '%(path)s' -> '%(ln_target)s' est un lien symbolique "
"circulaire."

#: src/sbackup/util/filecollect.py:388
#, python-format
msgid "File '%(file)s' is backup's target directory."
msgstr ""
"Le fichier '%(file)s' est le répertoire de destination de la sauvegarde."

#: src/sbackup/util/filecollect.py:393
#, python-format
msgid "Path '%(file)s' defined in excludes list."
msgstr "Le chemin '%(file)s' est défini dans la liste des exclusions."

#: src/sbackup/util/filecollect.py:401
#, python-format
msgid "File '%(file)s' matches regular expression '%(regex)s'."
msgstr ""
"Le fichier '%(file)s' correspond à l'expression régulière '%(regex)s'."

#: src/sbackup/util/filecollect.py:419
#, python-format
msgid ""
"File '%(file)s' exceeds maximum file size ( %(filesize)s > %(maxsize)s)."
msgstr ""
"Le fichier '%(file)s' excède la taille maximale de fichier ( %(filesize)s > "
"%(maxsize)s)."

#: src/sbackup/util/filecollect.py:497
#, python-format
msgid "Error while checking directory '%(dir)s': %(error)s."
msgstr "Erreur lors de la vérification du répertoire '%(dir)s' : %(error)s."

#: src/sbackup/util/filecollect.py:548
msgid "Empty regular expression found. Skipped."
msgstr "Une expression régulière vide a été trouvée et ignorée."

#: src/sbackup/util/filecollect.py:554
#, python-format
msgid "Invalid regular expression ('%s') found. Skipped."
msgstr "L'expression régulière ('%s') invalide a été ignorée."

#: src/sbackup/util/filecollect.py:573
msgid "No directories to backup defined."
msgstr "Aucun répertoire de destination de la sauvegarde n'a été défini."

#: src/sbackup/util/lock.py:76
msgid "Another application instance is already running."
msgstr "Une autre instance de l'application est déjà en cours d'exécution."

#: src/sbackup/util/lock.py:78
msgid "Invalid lock file found. Is being removed."
msgstr "Fichier verrou invalide. En cours de suppression."

#: src/sbackup/util/lock.py:128 src/sbackup/util/lock.py:142
#, python-format
msgid "Unable to remove lock file: %s"
msgstr "Impossible de supprimer le fichier verrou : %s"

#: src/sbackup/util/lock.py:132 src/sbackup/util/lock.py:144
msgid "Unable to remove lock file: File not found."
msgstr ""
"Impossible de supprimer le fichier de verrouillage: Fichier non trouvé."

#: data/ui/sbackup-config.glade:11
msgid "Exclude file type"
msgstr "Exclure un type de fichier"

#: data/ui/sbackup-config.glade:26
msgid "Standard file types:"
msgstr "Types standards de fichiers"

#: data/ui/sbackup-config.glade:63
msgid "Custom defined file type with "
msgstr "Types de fichiers personnalisés définis par "

#: data/ui/sbackup-config.glade:93
msgid "file extension"
msgstr "extension de fichier"

#: data/ui/sbackup-config.glade:163
msgid "Exclude by regex"
msgstr "Exclure par regex (expression régulière)"

#: data/ui/sbackup-config.glade:180
msgid ""
"Enter a regular expression.\n"
"Files and directories matching this expression will be excluded from backup."
msgstr ""
"Entre une expression régulière.\n"
"Les fichiers et les répertoires correspondant à cette expression seront "
"exclus de la sauvegarde."

#: data/ui/sbackup-config.glade:257
msgid "Enter the remote target to include"
msgstr "Entrer une cible distante à inclure"

#: data/ui/sbackup-config.glade:278
msgid "Choose a plugin to get the format"
msgstr "Choisir le plugin pour connaitre le format"

#: data/ui/sbackup-config.glade:295
msgid "Choose your plugin :"
msgstr "Choisir son plugin"

#: data/ui/sbackup-config.glade:367
msgid "Check"
msgstr "Vérifier"

#: data/ui/sbackup-config.glade:479
msgid ""
"<b>Save changed configuration?</b>\n"
"\n"
"The configuration has changed. If you want to use the modified "
"configuration, you have to save it.\n"
"\n"
"Do you want to save the configuration now?"
msgstr ""
"<b>Sauvegarder la nouvelle configuration?</b>\n"
"\n"
"La configuration a été modifiée. Si vous désirez l'utiliser, vous devez la "
"sauvegarder.\n"
"\n"
"Désirez-vous sauvegarder la configuration?"

#: data/ui/sbackup-config.glade:572
msgid "Configuration Profile Manager"
msgstr "Configuration du gestionnaire de profil"

#: data/ui/sbackup-config.glade:594
msgid ""
"You can set up multiple configuration profiles which are processed when "
"performing a backup. Each profile has its own set of settings including "
"backup destination, backup format etc."
msgstr ""
"Vous pouvez configurer plusieurs profils de configuration qui sont traités "
"lors de l'exécution d'une sauvegarde. Chaque profil a son propre ensemble de "
"paramètres, dont la destination de sauvegarde, le format de la sauvegarde, "
"etc."

#: data/ui/sbackup-config.glade:613
msgid "<b>Configuration profiles</b>"
msgstr "<b>Profils de Configuration</b>"

#: data/ui/sbackup-config.glade:785
msgid ""
"<b>Add new profile</b>\n"
"\n"
"Please enter the new profile name and set the according checkbox in order to "
"enable resp. disable it."
msgstr ""
"<b>Ajouter un nouveau profil</b>\n"
"\n"
"Veuillez entrer le nom du nouveau profil et cocher la case afin de l'activer "
"ou de le désactiver."

#: data/ui/sbackup-config.glade:810
msgid "Profile name"
msgstr "Nom du profil"

#: data/ui/sbackup-config.glade:836
msgid "Enabled"
msgstr "Activé"

#: data/ui/sbackup-config.glade:915 data/ui/sbackup-restore.glade:7
msgid "Simple Backup Suite"
msgstr "Solution de sauvegarde « Simple Backup »"

#: data/ui/sbackup-config.glade:932
msgid "_File"
msgstr "_Fichier"

#: data/ui/sbackup-config.glade:971
msgid "Save as..."
msgstr "Enregistrer sous..."

#: data/ui/sbackup-config.glade:1008
msgid "_Tools"
msgstr "_Outils"

#: data/ui/sbackup-config.glade:1016 data/ui/sbackup-config.glade:1140
msgid "Set default settings..."
msgstr "Définir les paramètres par défaut..."

#: data/ui/sbackup-config.glade:1037
msgid "Profile Manager..."
msgstr "Gestionnaire de profil..."

#: data/ui/sbackup-config.glade:1058
msgid "Make backup now"
msgstr "Faire une sauvegarde maintenant"

#: data/ui/sbackup-config.glade:1079
msgid "_Help"
msgstr "Aid_e"

#: data/ui/sbackup-config.glade:1087
msgid "_View all help topics"
msgstr "So_mmaire"

#: data/ui/sbackup-config.glade:1103
msgid "_About this program"
msgstr "À _propos  de ce programme"

#: data/ui/sbackup-config.glade:1154
msgid "Reload configuration"
msgstr "Recharger la configuration"

#: data/ui/sbackup-config.glade:1168
msgid "Save configuration"
msgstr "Sauvegarder la configuration"

#: data/ui/sbackup-config.glade:1191
msgid "Make a backup now"
msgstr "Faire une sauvegarde maintenant"

#: data/ui/sbackup-config.glade:1214
msgid "Open the profile manager"
msgstr "Ouvrir le gestionnaire de profil"

#: data/ui/sbackup-config.glade:1215
msgid "Profiles"
msgstr "Profils"

#: data/ui/sbackup-config.glade:1253
msgid ""
"none\n"
"gzip\n"
"bzip2"
msgstr ""
"aucun\n"
"gzip\n"
"bzip2"

#: data/ui/sbackup-config.glade:1271
msgid ""
"Compression format\n"
"Default is set to uncompressed backups."
msgstr ""
"Format de compression\n"
"Par défaut, la sauvegarde s'effectue sans compression."

#: data/ui/sbackup-config.glade:1290 data/ui/sbackup-config.glade:1403
msgid ""
"You can split the snapshot to make backup on some filesystem that doesn't "
"support large files. This functionality can not be used if the backup is "
"compressed."
msgstr ""
"Vous pouvez scinder l'instantané afin d'effectuer une sauvegarde dans un "
"système de fichier ne supportant pas les gros fichiers. Cette fonction ne "
"peut être utilisée si la sauvegarde est compressée."

#: data/ui/sbackup-config.glade:1330
msgid "Do a full backup at least once every"
msgstr "Faire une sauvegarde complète au moins tous les"

#: data/ui/sbackup-config.glade:1339
msgid ""
"Unless a set number of days has passed since the last full backup, "
"incremental backups will be done."
msgstr ""
"À moins que depuis le nombre de jours renseignés une sauvegarde complète ait "
"été faite, une sauvegarde incrémentielle sera réalisée."

#: data/ui/sbackup-config.glade:1356 data/ui/sbackup-config.glade:3015
msgid "days"
msgstr "Jours"

#: data/ui/sbackup-config.glade:1370
msgid " "
msgstr "> "

#: data/ui/sbackup-config.glade:1380 data/ui/sbackup-config.glade:2323
msgid "MiB"
msgstr "Mio"

#: data/ui/sbackup-config.glade:1405
msgid "Split backup archives into several chunks"
msgstr "Scinder l'archive de sauvegarde en plusieurs parties."

#: data/ui/sbackup-config.glade:1438 data/ui/sbackup-config.glade:2627
#: data/ui/sbackup-config.glade:2863 data/ui/sbackup-config.glade:3111
#: data/ui/sbackup-config.glade:3856
msgid "Help"
msgstr "Aide"

#: data/ui/sbackup-config.glade:1442 data/ui/sbackup-config.glade:2632
msgid "Show help topic about configuration GUI."
msgstr ""
"Voir la rubrique d'aide sur la configuration de l'interface graphique."

#: data/ui/sbackup-config.glade:1475
msgid "General"
msgstr "Général"

#: data/ui/sbackup-config.glade:1495
msgid "<b>Included files and directories</b>"
msgstr "<b>Fichier et répertoires inclus</b>"

#: data/ui/sbackup-config.glade:1570 data/ui/sbackup-config.glade:1818
#: data/ui/sbackup-config.glade:2023 data/ui/sbackup-config.glade:2181
msgid "Remove"
msgstr "Supprimer"

#: data/ui/sbackup-config.glade:1620 data/ui/sbackup-config.glade:1868
msgid "Add File"
msgstr "Ajouter un Fichier"

#: data/ui/sbackup-config.glade:1671 data/ui/sbackup-config.glade:1919
msgid "Add Directory"
msgstr "Ajouter un Répertoire"

#: data/ui/sbackup-config.glade:1718
msgid "Include"
msgstr "Inclure"

#: data/ui/sbackup-config.glade:1739
msgid "<b>Excluded files and directories</b>"
msgstr "<b>Fichier et répertoires exclus</b>"

#: data/ui/sbackup-config.glade:1952
msgid "Paths"
msgstr "Chemins"

#: data/ui/sbackup-config.glade:2074 data/ui/sbackup-config.glade:2232
msgid "Add"
msgstr "Ajouter"

#: data/ui/sbackup-config.glade:2109
msgid "File types"
msgstr "Types de fichiers"

#: data/ui/sbackup-config.glade:2267
msgid ""
"Regular\n"
"Expressions"
msgstr ""
"Expressions\n"
"Régulières"

#: data/ui/sbackup-config.glade:2288
msgid "Do not backup files bigger than"
msgstr "Ne pas sauvegarder les fichiers plus gros que"

#: data/ui/sbackup-config.glade:2344
msgid "Follow symbolic links"
msgstr "Suivre les liens symboliques"

#: data/ui/sbackup-config.glade:2373
msgid "Others"
msgstr "Autres"

#: data/ui/sbackup-config.glade:2400
msgid "Exclude"
msgstr "Exclure"

#: data/ui/sbackup-config.glade:2421
msgid "Use default backup directory"
msgstr "Utiliser le dossier de sauvegarde par défaut"

#: data/ui/sbackup-config.glade:2446
msgid "/var/backup"
msgstr "/var/backup"

#: data/ui/sbackup-config.glade:2472
msgid "Use custom local backup directory"
msgstr "Utiliser un répertoire de sauvegarde local personnalisé"

#: data/ui/sbackup-config.glade:2500
msgid "Select the destination folder"
msgstr "Choisir le répertoire de destination"

#: data/ui/sbackup-config.glade:2526
msgid "Use a remote site"
msgstr "Utiliser un site distant"

#: data/ui/sbackup-config.glade:2568
msgid "ssh://username:password@example.com/remote/dir/"
msgstr "ssh://nomdutilisateur:motdepasse@exemple.fr/distant/repertoire/"

#: data/ui/sbackup-config.glade:2580
msgid "Connect..."
msgstr "Se connecter..."

#: data/ui/sbackup-config.glade:2672
msgid ""
"<i>The selected destination is not writeable with current permissions. "
"Backups cannot be stored there.</i>"
msgstr ""
"<i>La destination choisie n'est pas accessible en écriture avec les droits "
"actuels. Les sauvegardes ne peuvent y être stockées.</i>"

#: data/ui/sbackup-config.glade:2704
msgid "Destination"
msgstr "Destination"

#: data/ui/sbackup-config.glade:2725
msgid "_No scheduled backups"
msgstr "_Aucune sauvegarde programmée"

#: data/ui/sbackup-config.glade:2742
msgid "_Simple"
msgstr "_Simple"

#: data/ui/sbackup-config.glade:2747
msgid ""
"This option will make a backup not at a precise time but when the computer "
"is up. "
msgstr ""
"Cette option fera une sauvegarde à un moment opportun lorsque l'ordinateur "
"est en fonction. "

#: data/ui/sbackup-config.glade:2761
msgid "_Custom"
msgstr "_Personnalisé"

#: data/ui/sbackup-config.glade:2781
msgid "Do backups"
msgstr "Faire les sauvegardes"

#: data/ui/sbackup-config.glade:2798
msgid "Cron time definition"
msgstr "Entrée cron"

#: data/ui/sbackup-config.glade:2834
msgid ""
"hourly\n"
"daily\n"
"weekly\n"
"monthly"
msgstr ""
"heure\n"
"quotidienne\n"
"hebdomadaire\n"
"mensuelle"

#: data/ui/sbackup-config.glade:2867
msgid "Show help topic about scheduled backups."
msgstr "Afficher l'aide à propos des sauvegardes planifiées"

#: data/ui/sbackup-config.glade:2905
msgid ""
"<small>You can configure scheduled (i.e. automated) backups either simply by "
"choosing an interval or precisely by a custom cron definition. See \"man 5 "
"crontab\" for cron time definition format.</small>"
msgstr ""
"<small>Vous pouvez planifier le calendrier de sauvegarde afin d'effectuer "
"des archives à intervalles réguliers ou plus finement en personnalisant une "
"tâche cron. Consulter \"man 5 crontab\" pour connaitre le format des entrées "
"cron.</small>"

#: data/ui/sbackup-config.glade:2937
msgid "Schedule"
msgstr "Planification"

#: data/ui/sbackup-config.glade:2955
msgid "Enable purging of old backups"
msgstr "Activer la purge des anciennes sauvegardes"

#: data/ui/sbackup-config.glade:2983
msgid "Simple cut-off: Erase all backups older than"
msgstr "Simple: Effacer toutes les archives plus vieilles que"

#: data/ui/sbackup-config.glade:3039
msgid "Logarithmic"
msgstr "Logarithmique"

#: data/ui/sbackup-config.glade:3065
msgid ""
"<small>Keep progressively less backups into the past:\n"
"Keep all backups from yesterday\n"
"Keep one backup per day from last week\n"
"Keep one backup per week from last month\n"
"Keep one backup per month from last year\n"
"Keep one backup per quarter from 2nd last year\n"
"Keep one backup per year further into past</small>"
msgstr ""
"<small>Conserver de moins en moins d'anciennes sauvegardes\n"
"Conserver toutes les sauvegardes de la veille\n"
"Conserver une sauvegarde par jour de la semaine dernière\n"
"Conserver une sauvegarde par semaine du mois dernier\n"
"Conserver une sauvegarde par mois de l'année dernière\n"
"Conserver une sauvegarde par trimestre de l'avant dernière année\n"
"Conserver une sauvegarde par année antérieure</small>"

#: data/ui/sbackup-config.glade:3116
msgid "Show help topic about purging of backups."
msgstr "Voir la rubrique d'aide sur le nettoyage des sauvegardes."

#: data/ui/sbackup-config.glade:3152
msgid ""
"<small>Note: Additionally all incomplete and irrecoverable backups will be "
"erased as long as there is a complete backup snapshot made later on.</small>"
msgstr ""
"<small>Note : De plus, toutes les sauvegardes non terminées ou non "
"récupérables seront effacées tant qu'il y a une sauvegarde complète faite "
"plus tard.</small>"

#: data/ui/sbackup-config.glade:3184
msgid "Purging"
msgstr "Nettoyage"

#: data/ui/sbackup-config.glade:3186
msgid "Cleaning old backups"
msgstr "Nettoyage des anciennes sauvegardes"

#: data/ui/sbackup-config.glade:3209
msgid "<b>Logging</b>"
msgstr "<b>Journalisation</b>"

#: data/ui/sbackup-config.glade:3234
msgid "Debug is just for debbuging purpose.Info is the recommended setting"
msgstr "Debug est juste pour le déboguage.Info est la valeur recommandée"

#: data/ui/sbackup-config.glade:3236
msgid "Log level"
msgstr "Niveau du journal"

#: data/ui/sbackup-config.glade:3248
msgid "Debug is just for debbuging purpose. Info is the recommended setting."
msgstr ""
"Debug est uniquement pour le déboguage. Info est la valeur recommandée"

#: data/ui/sbackup-config.glade:3274
msgid "Log file directory"
msgstr "Répertoire du fichier journal"

#: data/ui/sbackup-config.glade:3320
msgid "<b>Report by mail</b>"
msgstr "<b>Rapport via courriel</b>"

#: data/ui/sbackup-config.glade:3350
msgid "From"
msgstr "De"

#: data/ui/sbackup-config.glade:3361
msgid "SMTP server"
msgstr "Serveur SMTP"

#: data/ui/sbackup-config.glade:3375 data/ui/sbackup-config.glade:4053
msgid "Port"
msgstr "Port"

#: data/ui/sbackup-config.glade:3391
msgid "To"
msgstr "À"

#: data/ui/sbackup-config.glade:3478
msgid "Use a username and password"
msgstr "Utiliser un nom d'utilisateur et un password"

#: data/ui/sbackup-config.glade:3508 data/ui/sbackup-config.glade:4075
msgid "Username"
msgstr "Nom d'utilisateur :"

#: data/ui/sbackup-config.glade:3540 data/ui/sbackup-config.glade:4124
msgid "Password"
msgstr "Mot de passe :"

#: data/ui/sbackup-config.glade:3584
msgid "Use secure connection"
msgstr "Utiliser une connexion sécurisée"

#: data/ui/sbackup-config.glade:3615
msgid "Choose a certificate file"
msgstr "Choisissez un fichier de certificat"

#: data/ui/sbackup-config.glade:3629
msgid "Key"
msgstr "Clé"

#: data/ui/sbackup-config.glade:3645
msgid "Certificate"
msgstr "Certificat"

#: data/ui/sbackup-config.glade:3659
msgid "Choose a key file"
msgstr "Choisissez un fichier de clés"

#: data/ui/sbackup-config.glade:3731
msgid "Test _mail settings"
msgstr "Tester les réglages courriel"

#: data/ui/sbackup-config.glade:3772
msgid "Report"
msgstr "Rapport"

#: data/ui/sbackup-config.glade:3811
msgid "Warning"
msgstr "Avertissement"

#: data/ui/sbackup-config.glade:3860
msgid "Show help topic about default values."
msgstr "Voir la rubrique d'aide sur les valeurs par défaut."

#: data/ui/sbackup-config.glade:3893
msgid "_Set"
msgstr "Définir"

#: data/ui/sbackup-config.glade:3935
msgid "Set remote target"
msgstr "Définir la cible distante"

#: data/ui/sbackup-config.glade:3957
msgid "Type of service"
msgstr "Type de service"

#: data/ui/sbackup-config.glade:3997
msgid "Hostname"
msgstr "Serveur"

#: data/ui/sbackup-config.glade:4027
msgid "<b>Details</b>"
msgstr "<b>Détails</b>"

#: data/ui/sbackup-config.glade:4157
msgid "Show password"
msgstr "Afficher le mot de passe"

#: data/ui/sbackup-config.glade:4217
msgid "_Connect"
msgstr "Se _connecter"

#: data/ui/sbackup-config.glade:4293
msgid "Make a full backup"
msgstr "Faire une sauvegarde complète"

#: data/ui/sbackup-legacy-restore.glade:7
msgid "Restore files/directories"
msgstr "Restaurer les fichiers/répertoires"

#: data/ui/sbackup-legacy-restore.glade:25
msgid "<b>Restore Source Folder</b>"
msgstr "<b>Restaurer le répertoire d'origine</b>"

#: data/ui/sbackup-legacy-restore.glade:90
msgid "Available backups:"
msgstr "Sauvegardes disponibles :"

#: data/ui/sbackup-legacy-restore.glade:109
msgid ""
"Type in this box text like that:\n"
"\n"
"/home/user/folder - To use a local folder\n"
"ssh://user:password@machine:/remote/folder - To use a ssh location\n"
"ftp://user:password@machine:/remote/folder - To use a ftp location"
msgstr ""

#: data/ui/sbackup-legacy-restore.glade:179
msgid "Use custom:"
msgstr "Utiliser les réglages personnalisés :"

#: data/ui/sbackup-legacy-restore.glade:200
msgid "Use default:"
msgstr "Utiliser les réglages par défaut :"

#: data/ui/sbackup-legacy-restore.glade:241
msgid "<b>Files and Folders to restore</b>"
msgstr "<b>Fichiers et répertoires à restaurer</b>"

#: data/ui/sbackup-legacy-restore.glade:345 data/ui/sbackup-restore.glade:782
msgid "Restore"
msgstr "Restaurer"

#: data/ui/sbackup-legacy-restore.glade:390
msgid "Restore As..."
msgstr "Restaurer en tant que..."

#: data/ui/sbackup-legacy-restore.glade:448
msgid "Restoring, this might take some time ..."
msgstr ""

#: data/ui/sbackup-legacy-restore.glade:463
msgid "gtk-cancel"
msgstr ""

#: data/ui/sbackup-restore.glade:38
msgid "<b>Backup destination</b>"
msgstr "<b>Destination de la sauvegarde<\\b>"

#: data/ui/sbackup-restore.glade:52
msgid "Displays the manual in Help browser."
msgstr "Affiche le manuel dans le navigateur d'aide."

#: data/ui/sbackup-restore.glade:85
msgid "Default profile's destination"
msgstr "Destination du profil par défaut"

#: data/ui/sbackup-restore.glade:90
msgid ""
"Set backup destination to the place specified in default configuration "
"profile"
msgstr ""
"Définir la destination de la sauvegarde comme celle indiquée dans le profil "
"de configuration par défaut"

#: data/ui/sbackup-restore.glade:137
msgid "Custom destination"
msgstr "Destination personnalisée"

#: data/ui/sbackup-restore.glade:143
msgid ""
"Select a backup destination on your harddisk or at a remote site. Press "
"Apply when ready."
msgstr ""
"Choisissez la destination de sauvegarde sur votre disque dur ou sur un site "
"distant. Appuyez ensuite sur Appliquer."

#: data/ui/sbackup-restore.glade:184
msgid "Choose a destination folder. For remote sites fill URI in text field"
msgstr ""
"Choisissez un dossier de destination. Pour les sites distants, renseignez "
"l'URI dans le champ de texte"

#: data/ui/sbackup-restore.glade:201
msgid "Change destination to path specified in left text field"
msgstr ""
"Changer la destination par le chemin spécifié dans le champ de texte à gauche"

#: data/ui/sbackup-restore.glade:266
msgid "Select a bold printed date to get the list of backups for that date."
msgstr ""
"Choisissez une date en gras pour obtenir la liste des sauvegardes à cette "
"date."

#: data/ui/sbackup-restore.glade:310
msgid "Available backups"
msgstr "Sauvegardes disponibles"

#: data/ui/sbackup-restore.glade:398
msgid "Delete snapshot"
msgstr "Supprimer l'instantané"

#: data/ui/sbackup-restore.glade:436
msgid "Delete"
msgstr "Supprimer"

#: data/ui/sbackup-restore.glade:473
msgid "<b>Snapshot History</b>"
msgstr "<b>Historique des instantanés</b>"

#: data/ui/sbackup-restore.glade:499
msgid "Current base:"
msgstr "Base active:"

#: data/ui/sbackup-restore.glade:540
msgid "Select a snapshot on this list to rebase on it."
msgstr "Sélectionner l'instantané dans la liste pour l'appliquer comme base."

#: data/ui/sbackup-restore.glade:615
msgid "Rebase"
msgstr "Changer la base"

#: data/ui/sbackup-restore.glade:666
msgid "<b>Snapshot Management</b>"
msgstr "<b>Gestion de l'instantané</b>"

#: data/ui/sbackup-restore.glade:705
msgid "Snapshot details"
msgstr "Détails de l'instantané"

#: data/ui/sbackup-restore.glade:770 data/ui/sbackup-restore.glade:787
msgid "Restores files/directories from the selected snapshot"
msgstr "Restaurer les fichiers/répertoires de l'instantané sélectionné"

#: data/ui/sbackup-restore.glade:802 data/ui/sbackup-restore.glade:821
msgid ""
"Restores files/directories recursively starting from the base snapshot to "
"the selected snapshot"
msgstr ""
"Restaurer les fichiers / répertoires de manière récursive à partir de "
"l'instantané de base vers l'instantané sélectionné"

#: data/ui/sbackup-restore.glade:816
msgid "Revert"
msgstr "Rétrograder"

#: data/ui/sbackup-restore.glade:859
msgid "<b>Restoration Management</b>"
msgstr "<b>Gestion de la restoration</b>"

#: data/ui/sbackup-restore.glade:884
msgid "<b>Export Management</b>"
msgstr "<b>Gestion de l'exportation</b>"

#: data/ui/sbackup-restore.glade:976
msgid "Headline"
msgstr "Titre"

#: data/ui/sbackup-restore.glade:989
msgid "Content"
msgstr "Contenu"

#, python-format
#~ msgid "No need to rebase a full snapshot '%s'"
#~ msgstr ""
#~ "Il n'est pas nécessaire de changer la base d'une sauvegarde complète '%s'"

#, python-format
#~ msgid ""
#~ "'%(snapshotToRebase)s'  doesn't have 'base' file , it might have been broken "
#~ msgstr ""
#~ "%(snapshotToRebase)s' n'a pas le fichier 'base'. Il se pourrait qu'il soit "
#~ "cassé. "

#, python-format
#~ msgid ""
#~ "Cannot rebase a snapshot on an earlier one : '%(snapshotToRebase)s' <= "
#~ "'%(NewBaseSnapshot)s' "
#~ msgstr ""
#~ "Nous ne pouvons pas changer la base d'une sauvegarde à une sauvegarde "
#~ "postérieure : '%(snapshotToRebase)s' <= '%(NewBaseSnapshot)s' "

#, python-format
#~ msgid "Rebasing '%(current)s' on '%(base)s'"
#~ msgstr "Changement de la base de '%(current)s' à '%(base)s'"

#, python-format
#~ msgid "An error occurred when pulling snapshot '%s'."
#~ msgstr ""
#~ "Une erreur s'est produite lors de la récupération de l'instantané '%s'."

#, python-format
#~ msgid "Upgrading from v1.0 to v1.2: %s"
#~ msgstr "Mise à jour de v1.0 à v1.2 : %s"

#, python-format
#~ msgid "Upgrading to v1.3: %s"
#~ msgstr "Mise à jour à v1.3 : %s"

#, python-format
#~ msgid "Upgrading to v1.4: %s"
#~ msgstr "Mise à jour à v1.4 : %s"

#, python-format
#~ msgid "Upgrading to v1.5: %s"
#~ msgstr "Mise à jour à v1.5 : %s"

#, python-format
#~ msgid "Do you really want to rebase '%(current)s' on '%(base)s' ?"
#~ msgstr "Voulez vous vraiment changer la base de '%(current)s' à '%(base)s' ?"

#, python-format
#~ msgid "File '%(file)s' does not exist."
#~ msgstr "Le fichier '%(file)s' n'existe pas."

#~ msgid "Upgrade"
#~ msgstr "Mettre à jour"

#~ msgid ""
#~ "There are snapshots with old snapshot format. Please upgrade these using the "
#~ "Restoration tool if you want to use them."
#~ msgstr ""
#~ "Il y a des instantanés comportant un ancien format. Veuillez les mettre à "
#~ "niveau en utilisant l'utilitaire de Restauration si vous voulez les utiliser."

#, python-format
#~ msgid "Base of snapshot '%s' is not set."
#~ msgstr "La base de l'instantané '%s' n'est pas réglée,"

#, python-format
#~ msgid "Cancelling pull of snapshot '%s'"
#~ msgstr "Abandon de l'export du contenu de l'instantané '%s'"

#, python-format
#~ msgid "Unable to remove upgrade stamp: %s"
#~ msgstr "Impossible de supprimer la marque de la mise à jour: %s"

#, python-format
#~ msgid "The SNAR file already exist for snapshot '%s'. It is not overwritten."
#~ msgstr ""
#~ "Le fichier SNAR existe déjà pour l'instantané '%s'. Il ne sera pas écrasé."

#~ msgid ""
#~ "There are snapshots stored in outdated snapshot formats. Please upgrade them "
#~ "using 'Simple Backup-Restoration' if you want to use them."
#~ msgstr ""
#~ "Il y a des instantanés utilisant un format obsolète. Pour les utiliser, "
#~ "faire une mise à jour de ceux-ci en utilisant  'Simple Backup-Restoration'."

#~ msgid "Upgrade snapshot"
#~ msgstr "Mise à jour de l'instantané"

#~ msgid "Do you really want to upgrade snapshot?"
#~ msgstr "Voulez-vous vraiment mettre à jour l'instantané?"

#~ msgid "Snapshot was sucessfully upgraded."
#~ msgstr "L'instantané a été mise à jour avec succès."

#, python-format
#~ msgid ""
#~ "Snapshot version is not supported (Only %(supportedversion)s is supported). "
#~ "Version '%(currentversion)s' found. You should upgrade it. "
#~ msgstr ""
#~ "La version de l'instantané n'est pas prise en charge (uniquement "
#~ "%(supportedversion)s est supportée). La version actuel est "
#~ "'%(currentversion)s'. Vous devriez le mettre à jour. "

#~ msgid "The version of the snapshot is greater than the supported one!"
#~ msgstr "La version de l'instantané est supérieure à celle supportée."

#, python-format
#~ msgid ""
#~ "Upgrading the selected snapshot `%s` might corrupt your backup for this "
#~ "particular point in time. You cannot undo this operation. It is recommended "
#~ "to create a copy of the snapshot directory before upgrading it. The upgrade "
#~ "process takes a while - be patient.\n"
#~ "\n"
#~ "Are you sure that you want to upgrade the snapshot now?"
#~ msgstr ""
#~ "La mise à niveau de l'instantané sélectionné '%s' peut corrompre votre "
#~ "sauvegarde pour ce point précis dans le temps. Cette opération est "
#~ "irréversible. Il est recommandé de créer une copie du répertoire de "
#~ "l'instantané avant la mise à niveau. Le processus de mise à niveau prend un "
#~ "certain temps - soyez patient.\n"
#~ "\n"
#~ "Êtes-vous certain que vous souhaitez mettre à niveau l'instantané maintenant?"

#, python-format
#~ msgid "Upgrade snapshot '%s'"
#~ msgstr "Mise à jour de l'instantané '%s'"

#~ msgid "Simple Backup upgrade"
#~ msgstr "Mise à niveau de Simple Backup"

#~ msgid "Upgrade finished"
#~ msgstr "La mise à niveau est terminée"

#, python-format
#~ msgid ""
#~ "While attempting to upgrade snapshot the following error occurred:\n"
#~ "%s"
#~ msgstr ""
#~ "Lors de la mise à jour de l'instantané, l'erreur suivante est arrivée :\n"
#~ "%s"

#~ msgid "Unable to upgrade snapshot"
#~ msgstr "Échec de la mise à jour de l'instantané."
